indico/modules/events/registration/util.py

```diff
@@ -235,6 +236,7 @@ def create_registration(regform, data, invitation=None, management=False, notify
     registration.sync_state(_skip_moderation=skip_moderation)
     db.session.flush()
     notify_registration_creation(registration, notify_user)
+    signals.event.registration_created.send(registration, management=management)
```

Comment:
I would send the signal one line above, i.e. before sending notification emails.